homeassistant/components/ozw/strings.json

```
@@ -1,11 +1,21 @@
 {
   "config": {
     "step": {
-      "user": {
-        "title": "Confirm set up"
+      "on_supervisor": {
+        "title": "Select connection method",
+        "description": "Select how to connect to the OpenZWave controller MQTT broker",
```

Instead of talking about to connect, we should ask: Do you want to use the OZW add-on or do you want to manually set up the OZW controller.

---

We still require the MQTT integration after this PR. The internal connection etc will come in follow up PRs.
```suggestion
        "description": "Do you want to use the OpenZWave Supervisor add-on?",
```